### PR TITLE
Feature/change order series anime

### PR DESCRIPTION
**Pull request by p0psicles**
- [x] PR is based on the DEVELOP branch
- [x] Don't send big changes all at once. Split up big PRs into multiple smaller PRs that are easier to manage and review
- [x] Read the [contribution guide]

Screeny:
image

IT's WIP, but the finale version will get a small control in front of the list title, which you can drag to change the order.

Note, will need a rebase to get rid of some of the commits.

### CODE REVIEW

**Comment by p0psicles** [comment]
@OmgImAlexis i'm currently using jQuery sortable, with overwriting some of it's methods.
I'm planning to use the apiv2, to write back the sorted list each time the sorting finishes.

Thoughts?

**Comment by labrys** [comment]
@p0psicles I assume you want this so you can choose to sort by a name other than that listed under title.  Would adding a `Sort Title` field be helpful for you?

**Comment by p0psicles** [comment]
Sorry @labrys I don't understand. The goal of this Pr is to be able to have the anime on top. As I watch mainly anime. And I implemented it through a dragndrop.

**Comment by labrys** [comment]
@p0psicles ok.  I thought you were dragging and dropping the series themselves to change the sort order.

**Comment by p0psicles** [comment]
Haha no, although that would be cool.

**Comment by p0psicles** [comment]
But then we should add an order field to the tvshows table. Something for another time.

File: medusa/server/api/v2/config.py [hunk 1]
```diff
@@ -63,6 +64,7 @@ class ConfigHandler(BaseRequestHandler):
         'theme.name': StringField(app, 'THEME_NAME'),
         'backlogOverview.period': StringField(app, 'BACKLOG_PERIOD'),
         'backlogOverview.status': StringField(app, 'BACKLOG_STATUS'),
+        'layout.show.showListLayout': ListField(app, 'SHOW_LIST_ORDER'),
```
Comment (OmgImAlexis) [inline medusa/server/api/v2/config.py]: Should this be using `showListOrder` instead of `showListLayout`?

File: static/js/home/index.js [hunk 2]
```diff
@@ -365,4 +365,52 @@ MEDUSA.home.index = function() {
...
+            api.patch('config/main', {
+            layout: {
+                show: {
+                    showListLayout: showListOrder.toArray()
+                }
+            }
+            }).then(function(response) {
+                log.info(response);
+            }).catch(function(err) {
+                log.info(err);
```
Comment (OmgImAlexis) [inline static/js/home/index.js]: Change this to `.error`.

File: medusa/server/api/v2/config.py (hunk 1 above)
Comment (p0psicles) [reply medusa/server/api/v2/config.py]: Ow yeah. Tnx.

File: static/js/home/index.js (hunk 2 above)
Comment (p0psicles) [reply static/js/home/index.js]: Will do. Tnx

**Pull request merged**